dashboard/src/components/LazyInteractionDetails.tsx

```
@@ -0,0 +1,707 @@
+          );
+      }
+    } catch (error) {
+      return <DetailsErrorFallback error={error as Error} />;
+    }
+  };
+
+  // Create formatted text for copying based on interaction type
+  const getFormattedCopyText = () => {
+    switch (type) {
+      case 'llm':
+        const llmDetails = details as LLMInteraction;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)